tests/unit/tests_suite_test.go

```diff
@@ -16,6 +20,43 @@ const (
 	helmReleaseName             = "k8ssandra-test"
 )
 
+// Common labels for k8ssandra chart
+func GetK8ssandraRequiredLabels() map[string]string {
+
+	k8ssandraChart, err := chartutil.Load(chartsPath)
+	Expect(err).ToNot(BeNil())
+	Expect(k8ssandraChart).ToNot(BeNil())
+
+	meta := k8ssandraChart.Metadata
+	requiredLabels := map[string]string{
+		"helm.sh/chart":                "k8ssandra-" + meta.Version,
```

Comment:
Given the common labels are generated from k8ssandra-common, it would be good if we could simply get the output of `{{ k8ssandra-common.labels }}`.

Reply:
I agree, I will reference from there.